pkgs/applications/misc/golden-cheetah/default.nix

```diff
@@ -16,18 +16,18 @@ let
   };
 in mkDerivation rec {
   pname = "golden-cheetah";
-  version = "3.5";
+  version = "3.5-DEV1804-1240-gc9e01353a";
```

Comment:
It's kind of a tag (from git-describe man page)
`Otherwise, it suffixes the tag name with the number of additional commits on top of the tagged object and the abbreviated object name of the most recent commit. The result is a "human-readable" object name which can also be used to identify the commit to other git commands.`